core/invehicle-digital-twin/src/main.rs

```diff
@@ -61,6 +77,59 @@ async fn register_invehicle_digital_twin_service_with_chariott(
     Ok(())
 }
 
+/// Builds the enabled modules for the grpc server and starts the server.
+///
+/// # Arguments
+/// * `addr` - The address the server will be hosted on.
+/// * `base_service` - The core service that will be hosted.
+///
+/// # How to add an Module to this method:
+/// 1. Add a block of code with the appropriate cfg feature flag.
+/// 2. Create the `GrpcModule` object within the block - if applicable.
+/// 3. Create the `GrpcInterceptorLayer` object(s) within the block - if applicable.
+/// 4. Add the interceptors to the middleware stack with `.layer()`.
+/// 5. Call and return from the block `.add_module()` on the server with the updated middleware and
+/// module.
+#[allow(unused_assignments, unused_mut)] // Necessary when no extra modules are built.
+fn build_server_and_serve<S>(
+    addr: SocketAddr,
+    base_service: S,
+) -> impl Future<Output = Result<(), tonic::transport::Error>>
+where
+    S: Service<http::Request<Body>, Response = http::Response<BoxBody>, Error = Infallible>
+        + NamedService
+        + Clone
+        + Send
+        + 'static,
+    S::Future: Send + 'static,
+{
+    let mut server: GrpcServer<Identity> = GrpcServer::new(addr);
+
+    #[cfg(feature = "managed_subscribe")]
+    // (1) Adds the Managed Subscribe module to the service.
+    let server = {
+        // (2) Initialize the Managed Subscribe module, which implements GrpcModule.
+        let managed_subscribe_ext = ManagedSubscribeExt::new();
```

Comment:
Perhaps rename it to "managed_subscribe_module".

Reply:
And rename the associated struct name.

Reply:
updated here and elsewhere!